### PR TITLE
Integration testing data for features string char

### PR DESCRIPTION
**Pull request by avramidis**

### CODE REVIEW

**Comment by iglesias** [comment]
What test is using this data, @avramidis? I was looking for some mention to string/char in the file but could not find it :sweat_smile:

**Comment by avramidis** [comment]
Yes, you are correct. In this file it only shows the size of the char data. The example (shown below) that generated this data file is examples/meta/src/features/string_char.sg in https://github.com/shogun-toolbox/shogun/commit/f291e9c0879e97b594d0b2660df100fe30d4dc8f

The variables max_string_length number_of_strings are stored in the data file. The ideal could be to store in the data file the string_list. Do you know a way that I can do that? 

```
File words = csv_file("../../data/words.dat")

#![create_features]
Features f = string_features(words, enum EAlphabet.RAWBYTE)
#![create_features]

#![output stat]
int max_string_length = f.get_int("max_string_length")
int number_of_strings = f.get_int("num_vectors")
StringCharList string_list = f.get_char_string_list("string_list")
#![output stat]
```

File: testsuite/meta/features/string_char.dat [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<<_SHOGUN_SERIALIZABLE_ASCII_FILE_V_00_>>
+array Vector<SGSerializable*> 2 ({Serializable int32 [
+value int32 12
+]}{Serializable int32 [
```
Comment (karlnapf) [inline testsuite/meta/features/string_char.dat]: no need to add this here ... no content :)
Currently we only serialize float32/64 scalar/vector/matrix.
We can change that of course, but it needs some more headachy changes ... and no need to do that before we actually have an algorithm exposed that actually produces string data

Comment (avramidis) [reply testsuite/meta/features/string_char.dat]: OK, no problem. I will remove the data file.

**Pull request closed without merging**